stetho-repl-js/src/main/java/com/facebook/stetho/inspector/console/js/JavaScriptRuntimeRepl.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2014-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+package com.facebook.stetho.inspector.console.js;
+
+import com.facebook.stetho.common.LogUtil;
+import com.facebook.stetho.inspector.console.RuntimeRepl;
+import org.mozilla.javascript.Context;
+import org.mozilla.javascript.ScriptableObject;
+
+class JavaScriptRuntimeRepl implements RuntimeRepl {
+
+  private final ScriptableObject mJsScope;
+
+  JavaScriptRuntimeRepl(ScriptableObject scope) {
+    mJsScope = scope;
+  }
+
+  @Override
+  public Object evaluate(String expression) throws Throwable {
+      LogUtil.i("js) eval expression: %s", expression);
+
+      Object result;
+      final Context jsContext = enterJsContext();
+      try {
+        result = jsContext.evaluateString(mJsScope, expression, "chrome", 1, null);
+      } finally {
+        Context.exit();
+      }
+
+      LogUtil.i("js) Result: %s; type: %s", result, result == null ? null : result.getClass().getName());
```

Comment:
yeah, the logging through LogUtil is one of the first thing to land. Before I had access to the console all that I had was the logcat logs :)

I'm removing them.